common/modifiers/00_static_modifiers.txt

```diff
@@ -472,16 +472,15 @@ infamy_pariah = {
 # Scales with devastation level
 state_region_devastation = {
 	icon = gfx/interface/icons/timed_modifier_icons/modifier_gear_negative.dds
-	state_infrastructure_mult = -1.0
-	state_mortality_mult = 2.0
-	state_migration_pull_mult = -2.0
-	state_construction_mult = -1.0
-	state_tax_waste_add = 0.75
-	building_group_bg_agriculture_throughput_mult = -0.2
-    building_group_bg_ranching_throughput_mult = -0.8
-    building_group_bg_mining_throughput_mult = -0.8
-    building_group_bg_plantations_throughput_mult = -0.8
-    building_group_bg_manufacturing_throughput_mult = -0.8
+	 state_infrastructure_mult = -1.0
+	 state_mortality_mult = 2.0
+	 state_migration_pull_mult = -2.0
+	 state_construction_mult = -1.0
+	 state_tax_waste_add = 0.75
+	 building_group_bg_agriculture_throughput_mult = -0.2
+     building_group_bg_mining_throughput_mult = -0.8
+     building_group_bg_plantations_throughput_mult = -0.8
+     building_group_bg_manufacturing_throughput_mult = -0.8
```

Comment:
what is this formatting? using spaces and tabs.

Reply:
??? I don't have this formatting on VSC

Reply:
tabs and spaces, use one of them. not both.